src/com/google-map.js

```diff
@@ -125,28 +190,82 @@
 			}
 		};
 		
-		var init = function (p, selector) {
-			container = $(selector, p);
-			App.loaded(googleMap, function () {
-				initMap();
-			});
+		var loadDeps = function (ready) {
+			ready = ready || $.noop;
+			if (!window.isMapApiLoaded) {
```

Comment:
Why is this in window's namespace ?

Reply:
Because it needs to be super global if other modules/components wants to load the same deps. Do you have anything to propose ? I would use Symbol, but would still need a fallback...

Reply:
In that case i'm ok with this 😁